PyTorchDemo/PyTorchDemo/NLP/NLPPredictor.swift

```diff
@@ -10,25 +10,25 @@ class NLPPredictor: Predictor {
         }
     }()
 
-    private var topics: [String] = []
```

Comment:
why modifying this file? for the new machine translation demo, can we create a new one?

Reply:
All the changes in the PyTorchDemo folder were made and checked into my fork about a month ago while using the app to prototype the NMT app, and were, my bad, accidentally used for the PR.. . I'll close this PR, make changes based on your comments and create a new clean PR. Thanks!